catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/RemoteDeleteOperationsTest.java

```diff
@@ -268,8 +268,8 @@ private void assertEqualMetacards(String reason, List<Metacard> metacardList1,
                 .map(Metacard::getTitle)
                 .collect(Collectors.toList());
 
-        assertThat(reason, idList1, containsInAnyOrder(idList2));
-        assertThat(reason, titleList1, containsInAnyOrder(titleList2));
+        assertThat(reason, idList1, containsInAnyOrder(idList2.toArray()));
+        assertThat(reason, titleList1, containsInAnyOrder(titleList2.toArray()));
```

Comment:
This seems slightly odd (without having any background to why this is necessary). Why do we need to make the second argument to an array when the first is a list?